Scripts/travis.sh

```diff
@@ -108,7 +108,9 @@ elif [ "${TYPE}" == "EXAMPLE_PODS_SWIFT" ]; then
   cd gem/
   bundle install
   rake install
-  cd ..
+  cd ../Demo/EarlGreyExample
+  pod update
```

Comment:
If we don't add any new pods, pod update would work without a Pod install. Travis doesn't seem to clean when its done, which is why we kept getting the swift bug.